src/base/language_support.h

```diff
@@ -58,16 +58,27 @@
 
 #define LANG_IS_COERCIBLE_COLL(c)	\
   ((c) == LANG_COLL_ISO_BINARY || (c) == LANG_COLL_UTF8_BINARY	\
-   || (c) == LANG_COLL_EUCKR_BINARY)
+   || (c) == LANG_COLL_EUCKR_BINARY) || \
+  ((c) == LANG_COLL_ISO_BINARY_TS || (c) == LANG_COLL_UTF8_BINARY_TS \
+   || (c) == LANG_COLL_EUCKR_BINARY_TS)
+
+#define COLL_TS	15
+
+#define LANG_IS_SAME_COLLATION(c1,c2)   \
```

Comment:
how about LANG_IS_SAME_COLLATION => LANG_IS_SAME_EXCEPT_TS for readability.

Reply:
LANG_IS_SAME_COLLATION checks two collations are equal or similar including TS-variation.
I'm thinking the naming for more readability.